src/pages/Category/FilterBtn.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+
+class FilterBtn extends Component {
+  render() {
+    return (
+      <button
+        className={
+          this.props.currentFilter === `${this.props.filter}`
+            ? 'yellow-btn filter-btn'
+            : 'gray-btn filter-btn'
+        }
+        onClick={e => {
+          this.props.changeFilter(e.currentTarget.id);
+        }}
+        id={this.props.filter}
+      >
+        {this.props.text}
+      </button>
```

Comment:
```suggestion
        className={`filter-btn ${
          currentFilter === filter ? yellow-btn : gray-btn 
        }`}
        onClick={e => changeFilter(e.currentTarget.id)}
        id={filter}
      >
        {text}
      </button>
```
- 구조분해 할당 해주세요
- 클래스 셀렉터에 삼항연산자 사용하실 때 고정된 부분은 제외하고 템플릿 리터럴 사용하여 해당 부분에 대해서만 조건 걸어줄 수도 있겠습니다
- 8번 줄에 불필요한 템플릿 리터럴이 있어서 삭제하셔도 될거 같아요

Reply:
네, 클래스명을 이렇게 동적으로 수정하는게 많이 낯서네요. 더 연습해야겠습니다. 수정했습니다.